src/content/reducers/profile-view.js

```diff
@@ -347,39 +353,39 @@ export const selectorsForThread = (threadIndex: ThreadIndex): SelectorsForThread
       getCallTreeFilters,
       CallTreeFilters.getCallTreeFilterLabels
     );
-    const getRangeFilteredThread: State => Thread = createSelector(
+    const getRangeFilteredThread = createSelector(
       getThread,
       getDisplayRange,
-      (thread: Thread, range: StartEndRange) => {
+      (thread: Thread, range: StartEndRange): Thread => {
         const { start, end } = range;
         return ProfileData.filterThreadToRange(thread, start, end);
       }
     );
-    const _getRangeFilteredThreadSamples: State => SamplesTable = createSelector(
+    const _getRangeFilteredThreadSamples = createSelector(
       getRangeFilteredThread,
-      thread => thread.samples
+      (thread: Thread): SamplesTable => thread.samples
```

Comment:
Initially I typed the arguments too, but as you'll see below I stopped doing it then. I think this provides little value if the "getter" functions have their return values properly typed. What do you think ?